.changeset/shaggy-ducks-suffer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-application-studio': patch
+---
+
+Add DND for class properties ([#303] https://github.com/finos/legend-studio/issues/303
```

Comment:
Github issue link should be in the form of
`([#303](https://github.com/finos/legend-studio/pull/303))`
You can preview the markdown to verify its correct

Reply:
ah i see! thanks change will be made